package-lock.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
157410b7a671d6845f2bd717d7eb4a19b1c73e6b

This entire commit is a bit weird, is there a reason this was done? Can I just remove the entire commit?